docs/source/index.rst

```diff
@@ -66,6 +66,8 @@ A (citable) PDF of the ASReview documentation can be found on `Zenodo <https://d
 
    extensions/overview_extensions
 
+   extensions/extension_development
```

Comment:
@Rensvandeschoot I need your help here. I think this belongs to the development section (see below).

Reply:
I've moved it to development. (Also to manage broken links)